src/hotspot/share/utilities/objectBitSet.hpp

```diff
@@ -39,7 +39,7 @@ class MemRegion;
  * allocated on-demand only, in fragments covering 64M heap ranges. Fragments are never deleted
  * during the lifetime of the ObjectBitSet. The underlying memory is allocated from C-Heap.
  */
-template<MEMFLAGS F>
+template<MemTag F>
```

Comment:
More parameter names needing update.

Reply:
I followed the local pattern, but we can change this later in a followup.